client/clientservice/include/client/clientservice/event_service.hpp

```diff
@@ -37,6 +37,8 @@ struct EventServiceMetrics {
  private:
   concordMetrics::Component metrics_component_;
 
+  // TODO - GILL - update_processing_duration and write_duration should be removed later, as histogram/snapshots are
```

Comment:
Nit: Name needs to be part of TODO?

Reply:
It's really a matter of style.
Actually, I put it here since I'm not the main owner, I usually don't put if I wrote/modified most of the module.